File: project1/percentage.mjs
import * as readline from 'node:readline/promises';
import{stdin as input, stdout as output} from 'node:process';
const userInput = readline.createInterface({input, output});

console.log('Met dit programma kan je eenvoudig je resultaat in % omzetten.')
console.log('---')

let behaaldePunt = parseFloat(await userInput.question('Wat is je resultaat? '));
let totaalAantalPunten = parseFloat(await userInput.question('Wat het maximum aantal punten voor deze toets? '));

let consoleOutput = behaaldePunt / (totaalAantalPunten / 10) * 10;

console.log('Je hebt ' + consoleOutput.toFixed(2) + '% behaald.');

process.exit();

File: project1/percentage.py
print("Met dit programma kan je eenvoudig je resultaat in % omzetten.")
print("---")

behaaldePunten = int(input("Wat is je resultaat? "))
totaalAantalPunten = int(input("Wat het maximum aantal punten voor deze toets? "))

consoleOutput = behaaldePunten / (totaalAantalPunten / 10) * 10

print('Je hebt ' + str(round(consoleOutput, 2)) + '% behaald.')